File: api-python/schemas/product.py
def productEntity(item) -> dict:
    return {
        "id": str(item["_id"]),
        "name": item["name"],
        "description": item["description"],
        "value": item["value"],
        "category": item["category"],
        "quantity": item["quantity"],
        "image": item["image"],
    }

def productsEntity(entity) -> list:
    return [productEntity(item) for item in entity]

File: frontend/src/app/interfaces/Product.ts
export interface IProduct {
  name: string,
  description: string,
  value: string,
  category: string,
  quantity: string,
  image: string,
  convert?: number,
  status?: string,
}

export interface IDolar {
  USDBRL:{
    code: string,
    codein: string,
    name: string,
    high: string,
    low: string,
    varBid: string,
    pctChange: string,
    bid: string,
    ask: string,
    timestamp: string,
    create_date: string
  }

}
